CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 Changelog is kept with respect to version 0.11 of Entropies.jl. From version v2.0 onwards, this package has been renamed to ComplexityMeasures.jl.
 
+## 2.2
+
+- Corrected documentation for `SymbolicPermutation`, `SymbolicAmplitudeAwarePermutation`,
+    and `SymbolicWeightedPermutation`, indicating that the outcome space is the set of
+    `factorial(m)` *permutations* of the integers `1:m`, not the rank orderings,
+    as was stated before.
```

Comment:
You added this, but there aren't any git diffs here on the files of the estimators.

Reply:
You're right. I wrote the changelog before actually doing the changes. Fixed now.